server/routes.js

```diff
@@ -121,6 +121,7 @@ module.exports = (app) => {
    * Groups.
    */
   app.post('/groups', aN.authenticateUserByJwt(), required('group'), groups.create); // Create a group. Option `role` to assign the caller directly (default to null).
+  app.post('/groups/create_from_github_repo', aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, required('payload'), connectedAccounts.createGroupFromGithubRepo)
```

Comment:
Moved it to groups (which required a little bit of juggling to avoid recursion in our controllers - we need to refactor them). Added an ifParam as described above.